janus-idp/backstage-showcase
# chore(wrapper): rhidp-4257 add topology wrapper

## Description

Added wrappers for backstage-community/plugin-topology and backstage-community/plugin-topology-common. Verified it exports the topology plugin statically and dynamically (with changes in BCP PR) with no issues.

## Which issue(s) does this PR fix

- Migrating topology plugin from backstage-plugins repo to BCP
- Blocked by: https://github.com/backstage/community-plugins/pull/1470
- ~Blocked by: https://github.com/backstage/community-plugins/pull/1723~
- Blocked by: https://github.com/backstage/community-plugins/pull/1827

## PR acceptance criteria

Please make sure that the following steps are complete:


## How to test changes / Special notes to the reviewer
1. Go to `dynamic-plugins/wrappers/backstage-community-plugin-topology/` directory
2. Run `yarn install` and `yarn export-dynamic` to see it runs both scripts with no error
